src/tequila/circuit/gates.py

```diff
@@ -42,10 +35,14 @@ def RotationGate(axis: int, angle: typing.Union[typing.Hashable, numbers.Number]
     -------
     QCircuit object with this RotationGate
     """
-    return QCircuit.wrap_gate(RotationGateImpl(axis=axis, angle=angle, target=target, control=control))
+    target = list_assignment(target)
+    gates  = [RotationGateImpl(axis=axis, angle=angle, target=q, control=control) for q in target]
+
+    return QCircuit.wrap_gate(gates)
 
 
-def PowerGate(name: str, target: typing.Union[list, int], power: bool = None, control: typing.Union[list, int] = None):
+def PowerGate(name: str, target: typing.Union[list, int], power: bool = None, control: typing.Union[list, int] = None,
```

Comment:
name can now be auto-assigned by the generator or vice versa

would allow that
give name a default: name=None and just incude some consistency checks below

if name is None and generator is None:
    raise TequilaException("name or generator needs to be provided to initialize a power gate")

if name is not None and generator is not None:
    #failsafe for standard gates
    if name.lower()=="x":
        assert generator == paulis.X(target)
    etc (would do for X, Y, Z and H)
    warnings.warn("name {} and generator {} provided to PowerGate".format(name, generator), TequilaWarning)